ctapipe/reco/stereo_combination.py

```diff
@@ -205,6 +208,55 @@ def _combine_classification(self, event):
         )
         event.dl2.stereo.classification[self.prefix] = container
 
+    def _combine_disp(self, event):
+        ids = []
+        alt_values = []
+        az_values = []
+        weights = []
+
+        for tel_id, dl2 in event.dl2.tel.items():
+            mono = dl2.geometry[self.prefix]
+            if mono.is_valid:
+                alt_values.append(mono.alt)
+                az_values.append(mono.az)
+                dl1 = event.dl1.tel[tel_id].parameters
+                weights.append(self._calculate_weights(dl1) if dl1 else 1)
+                ids.append(tel_id)
+
+        if len(alt_values) > 0:  # by construction len(alt_values) == len(az_values)
+            coord = AltAz(alt=alt_values, az=az_values)
+            # https://en.wikipedia.org/wiki/Von_Mises%E2%80%93Fisher_distribution#Mean_direction
+            mono_x, mono_y, mono_z = coord.cartesian.get_xyz()
+            stereo_x = np.average(mono_x, weights=weights)
+            stereo_y = np.average(mono_y, weights=weights)
+            stereo_z = np.average(mono_z, weights=weights)
+
+            mean_cartesian = CartesianRepresentation(x=stereo_x, y=stereo_y, z=stereo_z)
+            mean_spherical = mean_cartesian.represent_as(SphericalRepresentation)
+            mean_altaz = AltAz(mean_spherical)
+
+            # https://en.wikipedia.org/wiki/Directional_statistics#Measures_of_location_and_spread
+            r = mean_spherical.distance.to_value()
+            std = np.sqrt(-2 * np.log(r))
+
+            valid = True
+        else:
+            mean_altaz = AltAz(
+                alt=u.Quantity(np.nan, u.deg, copy=False),
+                az=u.Quantity(np.nan, u.deg, copy=False),
+            )
+            std = np.nan
+            valid = False
+
+        event.dl2.stereo.geometry[self.prefix] = ReconstructedGeometryContainer(
+            alt=mean_altaz.alt.to(u.deg),
+            az=mean_altaz.az.to(u.deg),
+            ang_distance_uncert=u.Quantity(np.rad2deg(std), u.deg, copy=False),
```

Comment:
do we need the `.to(u.deg)` or does the container handle `rad`? same for predict_table below

Reply:
THe container handles rad, it will be converted to the unit specified in the container when writing

Reply:
Does this also work in `predict_table`? Since it is never saved in a container in that case.